build-and-test-all-activemq.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash
+
+set -e
```

Comment:
This needs be invoked by CircleCI

Reply:
Also, rather than copy/paste build-and-test-all-mysql/postgres.sh, please implement a shared script _build-and-and-test.sh that's parameterized to remove duplication.